Gemfile.lock

```diff
@@ -104,10 +105,9 @@ GEM
       activesupport (>= 5.0.0)
     jwt (2.8.2)
       base64
-    listen (3.1.5)
+    listen (3.0.8)
```

Comment:
Mmmm we are actually downgrading the version of `listen` 🤔

Reply:
It seems that `listen 3.1.5` is not compatible with `Ruby > 3` versions.
<img width="219" alt="Screenshot 2024-09-06 at 4 05 01 PM" src="https://github.com/user-attachments/assets/2f3e8fe5-77bb-41a3-9442-3daaf58e6eef">